client/components/actions.js

```diff
@@ -0,0 +1,32 @@
+/* global _ */
+
+function searchGithubUsers (string) {
+  const url = 'https://api.github.com/search/users?q=' + string + '+in:login'
+  window.fetch(url)
+    .then(res => res.json())
+    .then(data => Session.set('searchUsers', data.items.map(item => item.login)))
+}
+
+Template.actions.helpers({
+  results: function () {
+    return Session.get('searchUsers')
+  }
+})
+
+Template.actions.events({
+  // search github users
+  'keyup .searchUser': _.debounce(function (event) {
+    searchGithubUsers(event.target.value)
+  }, 500),
```

Comment:
@calvintan you can use something like `_.debounce` or `_.throttle` on keyboard events to periodically save the text from editor to the database